Update README.md

razage opened the pull request:
typo

razage committed:
Update README.md

typo

crazybmanp reviewed (changes requested):
Don't include OWext, its name is not final.

razage committed:
removed examples

Quotes is the only valid example since OWext is going to be renamed. Even though it's not pythonic, I'm caving to the package having a capital letter. This also means that memes needs to be renamed.
